activestorage/lib/active_storage.rb

```diff
@@ -71,6 +71,10 @@ module ActiveStorage
 
   mattr_accessor :video_preview_arguments, default: "-y -vframes 1 -f image2"
 
+  def self.disk_service_enabled?
+    Blob.service&.is_a? Service::DiskService
+  end
```

Comment:
What happens if there's a mirror service configured and one of the mirrors is a disk service?

Reply:
Good point! We can probably handle this case by checking if the service is a mirror and if any of its mirrors/primary service is `Disk::Service`.

Reply:
@stefannibrasil hi! can you implement your suggestion?

Reply:
hi @guilleiguaran yes, we're working on it.

Reply:
should be good to review now! 🙌